deps/obs-scripting/cmake/python.cmake

```diff
@@ -7,6 +7,8 @@ if(ENABLE_SCRIPTING_PYTHON)
 
   if(OS_WINDOWS)
     find_package(Python 3.8...<3.11 REQUIRED Interpreter Development)
+  elseif(OS_LINUX)
```

Comment:
Probably should include FreeBSD and OpenBSD here as well as neither uses runtime linking.

Reply:
Done